vms/platformvm/state/stakers_ops_test.go

```diff
@@ -0,0 +1,506 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package state
+
+import (
+	"fmt"
+	"math"
+	"reflect"
+	"testing"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/set"
+	"github.com/leanovate/gopter"
+	"github.com/leanovate/gopter/gen"
+	"github.com/leanovate/gopter/prop"
+)
+
+// TestSimpleStakersOperations checks that State and Diff conform our stakersStorageModel.
+// TestSimpleStakersOperations tests State and Diff in isolation, over simple operations.
+// TestStateAndDiffComparisonToStorageModel carries a more involved verification over a production-like
+// mix of State and Diffs.
+func TestSimpleStakersOperations(t *testing.T) {
```

Comment:
Is this test testing that the Operations work as specified?

Reply:
Yes. Not sure how to improve the comment to clarify it.